### ISSUE TITLE
More items, smaller icons, less table padding on start page

### ISSUE THREAD

**Issue by Th0maz** [reloxx13/TasmoAdmin#325]
I'd like to see more items on the start page without scrolling on my mobile phone. 
Is there a way to rearrange the icons, add another row, display smaller icons or decrease the padding between items on this site? Which values do I have to edit in the .php file?

**Comment by reloxx13** [comment]
i added a custom.default.css
you can rename it to custom.css and write your own css

this wont work for docker right now (on an docker update the custom.css would be gone).
ill have to move it to the data dir to make it persistent in docker.

**Comment by reloxx13** [comment]
i made them a lil bit smaller, try the beta.

**Comment by Th0maz** [comment]
@reloxx13 I finally found the time to test the development version.
However the icons are larger than in the previous version...
![Screenshot_20191218-130652_before](https://user-images.githubusercontent.com/12065628/71085470-e5f46980-2197-11ea-8cd3-e60b68b0e029.png)
![Screenshot_20191218-130703_after](https://user-images.githubusercontent.com/12065628/71085471-e68d0000-2197-11ea-80a4-ecc417752d43.png)

**Comment by reloxx13** [comment]
Heya,

thanks for feedback, ill adjust it again.

I added more breakpoints already, but your screenshot is looking like its a few pixels too small to break to the smaller view.

If you know the pixel resultion of the device i could make a breakpoint for it and adjust it as needed.

**Comment by Th0maz** [comment]
With the latest 1.6.4-beta2 I'm back to 4 columns now on my mobile phone.
I'll have another look into your change sets and try to make it 5 or 6 columns for my environment.
![Screenshot_20191219-115121](https://user-images.githubusercontent.com/12065628/71168088-4518b380-2256-11ea-8d4a-710f6276d9b7.png)

**Comment by Th0maz** [comment]
> Is this still relevant? If so, what is blocking it? Is there anything you can do to help move it forward?
> 
> This issue has been automatically marked as stale because it has not had recent activity. It will be closed if no further activity occurs.

Yes, I still wish I could get more icons on a screen. Maybe 5 or 6 rows depending on a user-definable configuration for icon size or similar.
I just tried the latest version 3.0.2 and it's still 4 rows on my mobile devices.

**Comment by reloxx13** [comment]
Triage note: this is only partially addressed. TasmoAdmin already supports custom CSS overrides and there were historical mobile sizing tweaks, but there is still no built-in configurable icon density or compact layout for the start page. Keeping this open as a low-priority UI enhancement.
